client/components/jetpack/daily-backup-status/index.jsx

```diff
@@ -276,7 +276,7 @@ class DailyBackupStatus extends Component {
 			timezone: timezone,
 			gmtOffset: gmtOffset,
 		} );
-		const yesterday = today.subtract( 1, 'days' );
+		const yesterday = moment( today ).subtract( 1, 'days' );
```

Comment:
Probably here was the issue, with the `subtract` method we get yesterday but today change too, it mutates the today object state.

Reply:
Good catch.